### PR TITLE
CrestronElectronicsInc.CrestronToolbox version 3.1130.0019.0

### PR DESCRIPTION
**Pull request by botbeans**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98808)

### CODE REVIEW

File: manifests/c/CrestronElectronicsInc/CrestronToolbox/3.1130.0019.0/CrestronElectronicsInc.CrestronToolbox.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: CrestronElectronicsInc.CrestronToolbox
+PackageVersion: 3.1130.0019.0
```
Comment (botbeans) [inline manifests/c/CrestronElectronicsInc/CrestronToolbox/3.1130.0019.0/CrestronElectronicsInc.CrestronToolbox.installer.yaml:5]: InstallerType: "exe"
InstallModes:
- "/silent"

**Review by botbeans — COMMENTED** [review]
InstallerType: "exe"
InstallModes:

"/silent"

File: manifests/c/CrestronElectronicsInc/CrestronToolbox/3.1130.0019.0/CrestronElectronicsInc.CrestronToolbox.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/c/CrestronElectronicsInc/CrestronToolbox/3.1130.0019.0/CrestronElectronicsInc.CrestronToolbox.installer.yaml:5]: Hi @botbeans,

Was this comment supposed to be wrapped in "suggestion" marks? It seems like a good addition to this manifest.

**Comment by botbeans** [comment]
It actually was



Elvin Salcedo

Provisioning Manager

Support: 941.256.8700

Toll Free: 888.520.5229

Direct: 888.520.5229 ext. 2102

Website: AVIT.Partners<http://www.avit.partners/>

[cid:3c3b8cdf-da77-4316-8aa6-9e0877afdf49]
Confidentiality Notice: The information transmitted is intended only for the person or entity to which it is addressed and may contain material which is confidential, proprietary, privileged and/or otherwise legally exempt from disclosure. If you are not the intended recipient, any review, retention, copying, re-transmission, dissemination or other use of, or taking of any action in reliance upon this information by person or entities other than the intended recipient is prohibited.  If you received this in error, please contact the sender and delete the material and any attachments from your computer and network. There is no intent on the part of the sender to waive any privilege, including the attorney-client privilege.

________________________________

**Comment by stephengillie** [comment]
Hi @botbeans,

The package requires `Microsoft.VCRedist.2015+.x86` to install:
image

Pending:
* https://github.com/microsoft/winget-cli/issues/163

**Comment by stephengillie** [comment]
Hi @botbeans,

Is there a better InstallerUrl for this package?

**Pull request closed without merging**